### ISSUE TITLE
Misleading websites

### ISSUE THREAD

**Issue by Aiosa** [fengsp/flask-session#138]
Can you please communicate with the other version's (https://github.com/mcrowson/flask-sessionstore-fork) developers and kill or update (with some pointer/redirect...) https://flask-session.readthedocs.io/en/latest/? 

This 5-years-without-update web pops up in Google Search instead of the official documentation page, and even this Github repo is way down in the results, misleading the people. The afore-mentioned repository has been archived as you can see. Why keeping fossils around...  Thanks!

**Comment by yrro** [comment]
I've tried to contact the https://github.com/mcrowson/flask-sessionstore-fork maintainers but have received no response. There's nothing that the maintainers of this project can do about that.

I've also tried to contact the operators of Red the Docs, but haven't had any response from them either. To be fair to them it's not their job to sort out who should own https://readthedocs.org/projects/flask-session/ - projects are handed on a first come, first saved basis.
